esrally/mechanic/supplier.py

```diff
@@ -188,6 +195,10 @@ def _supply_requirements(sources, distribution, plugins, revisions, distribution
         supply_requirements["elasticsearch"] = ("distribution", _required_version(distribution_version), False)
 
     for plugin in plugins:
+        if plugin.moved_to_module:
+            # TODO: https://github.com/elastic/rally/issues/1622
+            continue
+
```

Comment:
I think this is a `pass` here, rather than a `continue`? (I mean there is no loop)

also a nit: shouldn't the line below be converted to an `elif`?

Reply:
> I think this is a pass here, rather than a continue? (I mean there is no loop)
I think `continue` is the correct choice here because it is inside a `for` loop.

> also a nit: shouldn't the line below be converted to an elif?
Addressed in  https://github.com/elastic/rally/pull/1624/commits/f6d36e010a31ac73b35c5e279b2880eebd2f6b76

Reply:
> I think `continue` is the correct choice here because it is inside a `for` loop.
> 
> >

Ah ofc missed that :+1: